gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Is a new version necessary?

Reply:
Yes it was. As https://docs.gradle.org/current/userguide/compatibility.html

Reply:
No, it isn't necessary :) In Gradle 8.4 is embedded Kotlin 1.9.10, but you can use Kotlin 1.9.10 with older versions of Gradle. But no problem, keep as is 👍